code/controllers/subsystem/shuttle.dm

```diff
@@ -69,13 +70,20 @@ SUBSYSTEM_DEF(shuttle)
 		supply_packs[P.type] = P
 		CHECK_TICK
 
-	// Initialize ftl13 station stocks
-	for(var/datum/star_system/system in SSstarmap.star_systems)
-		for(var/datum/planet/P in system.planets)
-			if(!P.station)
-				continue
-			P.station.generate()
-			CHECK_TICK
+// Initialize ftl13 station stocks/ station modules
+	var/list/module_weights = list()
+	for(var/i in SSstarmap.station_modules)
```

Comment:
Avoid this usage of the `in` keyword. Do `for(var/i in 1 to SSstarmap.station_modules.len)`